poly/constants.go

```diff
@@ -0,0 +1,15 @@
+package poly
+
+import (
+	"github.com/dedis/crypto/edwards"
+)
+
+const (
```

Comment:
I'd prefer getting this branch in shape and merged before putting in other things, mostly because the ```cothority``` also depends on it.

+1 for doing the threshold in a new branch, once this is accepted.